src/commonMain/kotlin/operations/numeric/unwrap/LenientUnwrapStrategy.kt

```diff
@@ -0,0 +1,25 @@
+package operations.numeric.unwrap
+
+import asList
+
+internal interface LenientUnwrapStrategy: UnwrapStrategy {
+    override fun unwrapValues(wrappedValue: Any?) = wrappedValue.asList.map(::unwrap)
```

Comment:
I wonder if the fact that this function unwraps to List<Double?> is hidden in implementation (neither class nor function name suggests it) is ok. I guess you'll create similar functions for other types for other operators?

Not a blocker

Reply:
In the future I will try to reuse these unwrappers as much as possible. This hidden implementation fact is bad in my opinion and might be tricky. I changed it as you said to keep it on the same abstraction level

Reply:
hmmm I thought you'll mark somehow that the operator that uses it needs casting to number (so unwrapValues will return List<Double> or maybe List<Number>). I'm not sure how returning `List<Any?>` makes this code better but maybe it'll be more clear when you add casting to other types 🤔